step-capstone/src/components/TravelObject.js

```diff
@@ -22,20 +22,24 @@ export default function TravelObject(props) {
     return (
         <Card>
             <CardContent>
-                <Grid container direction="row" justify="flex-end" alignItems="center">
+                <Grid container direction="row">
                     <Grid item xs>
                         <Box mr={10} width="100%" height="100%">
                             {content}
                         </Box>
                     </Grid>
                     <Grid item>
                         <Grid container direction="column">
-                            <Edit
-                                onClick={() => props.onEditItem(props.data.id)}
-                            />
-                            <Delete
-                                onClick={() => props.onRemoveItem(props.data.id)}
-                            />
+                            <Grid item>
```

Comment:
Just a question to confirm my understanding: to add scrolling functionality to itinerary container, we need all the travel object to be in grid? or what makes the scrolling functionality possible?

Reply:
I have it so that scrolling will automatically be enabled in the itinerary container if travelobjects continue past the given space (it's in the css)